src/ArrivalEvent.java

```diff
@@ -0,0 +1,26 @@
+import java.time.LocalTime;
+
+public class ArrivalEvent() {
+  int currentFloor;
+  LocalTime time;
+  Elevator elevator;
+
+  public ArrivalEvent(int currentFloor, LocalTime time, Elevator elevator) {
```

Comment:
@danihashweh @abdol99 you guys should all decide what time/date object you want to use to stay consistent

Reply:
We use Localtime as well